codec-memcache/src/test/java/io/netty/handler/codec/memcache/binary/BinaryMemcacheDecoderTest.java

```diff
@@ -182,7 +178,6 @@ public void shouldHandleTwoMessagesInOneBatch() {
         BinaryMemcacheRequest request = channel.readInbound();
         assertThat(request, instanceOf(BinaryMemcacheRequest.class));
         assertThat(request, notNullValue());
-        request.release();
```

Comment:
I'm not sure if it was broken before but undecteded, since the key was a string. after converting to bytebuf, all those releases are throwing bytebuf exceptions because of double-free. I think the embedded channel frees them on close, which happens after each unit test. but if you could double check that for sanity, it would be good